pyairtable/api/abstract.py

```diff
@@ -17,12 +23,21 @@ class ApiAbstract(metaclass=abc.ABCMeta):
     API_URL = posixpath.join(API_BASE_URL, VERSION)
     MAX_RECORDS_PER_REQUEST = 10
 
-    session: requests.Session
-    tiemout: Optional[Tuple[int, int]]
+    session: Session
+    tiemout: TimeoutTuple
+
+    def __init__(
+        self,
+        api_key: str,
+        timeout: Optional[TimeoutTuple] = None,
+        retry_strategy: Optional["compat.Retry"] = None,
+    ):
+
+        if not retry_strategy:
+            self.session = Session()
+        else:
+            self.session = _RetryingSession(retry_strategy)
```

Comment:
Then here we can do:

`
self.session = _RetryingSession(retry_strategy, prefixes=(self.API_URL,))
`